File: src/main/java/no/nav/service/pensjon/regler/to/FaktoromregnBeregningBatchRequest.java
package no.nav.service.pensjon.regler.to;

import java.util.Date;
import java.util.Vector;

import no.nav.domain.pensjon.regler.gomregning.BeregningerTilFaktoromregningGrunnlag;

public class FaktoromregnBeregningBatchRequest  extends ServiceRequest {

	private Date gammelGGjaldtDato;
	private Date nyGOmregnFraDato;
	private Vector<BeregningerTilFaktoromregningGrunnlag> beregningerTilFaktoromregningGrunnlagListe = new Vector<BeregningerTilFaktoromregningGrunnlag>();


}

File: src/main/kotlin/no/nav/pensjon/regler/to/FaktoromregnBeregningBatchRequest.kt
package no.nav.pensjon.regler.to

import no.nav.pensjon.regler.domain.gomregning.BeregningerTilFaktoromregningGrunnlag
import java.util.*
import java.time.LocalDate

class FaktoromregnBeregningBatchRequest : ServiceRequest() {
    var gammelGGjaldtDatoLd: LocalDate? = null
    var nyGOmregnFraDatoLd: LocalDate? = null
    var beregningerTilFaktoromregningGrunnlagListe = Vector<BeregningerTilFaktoromregningGrunnlag>()

}